packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts

```diff
@@ -29,8 +29,8 @@ export const AWS_CDK_METADATA = new Set([
   'me-south-1',
   // 'me-central-1',
   'sa-east-1',
-  // 'ap-south-2',
   // 'eu-central-2',
+  // 'ap-south-2',
```

Comment:
(Non-blocking) Side note: This change is a noop. We probably should order that list alphabetically.... 🙈